assets/Menu/settings.json

```diff
@@ -0,0 +1 @@
+{"window_name": "No Time to Spy", "resolution": "1920x1080", "frame_rate": 60, "audio_effects": 29.885057471264385, "audio_music": 66.0919540229885, "address": "10.24.12.33", "port": 13398}
```

Comment:
Was sind das für Zahlen (`audio_effects` etc)? Kommt das aus einem GUI Designer oder so?

Reply:
Sind das die default settings?

Reply:
Die werden gespeichert und geladen. Ist immer das was vorher eingegeben wurde.

Reply:
alright. Was ist mit window_name und frame_rate? Die kann man ja nicht selber einstellen. For now finde ich das ok, aber es wäre natürlich cool die user-config wo anders (`~/.config/...` z.B.) zu speichern